api_tests/e2e/rfc003/btc_eth-erc20/test.js

```diff
@@ -334,29 +301,29 @@ describe("RFC003: Bitcoin for ERC20", () => {
             );
         res.should.have.status(200);
         bob_redeem_action = res.body;
-
-        logger.info(
-            "Bob retrieved the following redeem parameters",
-            bob_redeem_action
-        );
     });
 
-    let bob_btc_balance_before;
-
     it("[Bob] Can execute the redeem action", async function() {
         bob_redeem_action.payload.should.include.all.keys("hex", "network");
-        bob_btc_balance_before = await bitcoin.btc_balance(bob_final_address);
-
         await bob.do(bob_redeem_action);
         await bitcoin.btc_generate();
     });
 
     it("[Bob] Should have received the alpha asset after the redeem", async function() {
-        let bob_btc_balance_after = await bitcoin.btc_balance(
+        this.timeout(10000);
+        let body = await bob.poll_comit_node_until(
+            chai,
+            bob_swap_href,
+            body => body.state.alpha_ledger.status === "Redeemed"
+        );
+        let bob_redeem_txid = body.state.alpha_ledger.redeem_tx;
+
+        let bob_satoshi_received = await bitcoin.getSatoshiTransferredTo(
```

Comment:
🔧 The case of `getSatoshiTransferredTo` is inconsistent with the_way_we_normally_write_things

Reply:
Well, Rust code style != JS code style ...

But in order to be consistent now, it is better to have snake_case.